src/raqm.c

```diff
@@ -2068,7 +2074,7 @@ _raqm_u8_to_u32_index (raqm_t   *rq,
 static size_t
 _raqm_count_codepoint_utf16_short (uint32_t chr)
 {
-  if (chr > 0x010000)
+  if (chr > 0x00010000)
```

Comment:
What is the point of this change?

Reply:
In hindsight it may have actually been pointless.

Reply:
Should be gone now :)